agroal-pool/src/main/java/io/agroal/pool/wrapper/ConnectionWrapper.java

```diff
@@ -141,6 +141,7 @@ public void releaseTrackedStatement(Statement statement) {
 
     @Override
     public void close() throws SQLException {
+        handler.traceConnectionOperation( "close" );
```

Comment:
Use method signature for all these, e.g.

```java
close()
abort(Executor)
setAutoCommit(boolean)
...
```

it could make it easier to track down the precise call stack